File: PF Liu/PF1.py
import math

while True:
	length=int(input())
	width=int(input())
	height=int(input())

	a=length*width
	b=width*height
	c=length*height

	print('the surface area is {} '.format(2*(a+b+c)))
	print('the volumn is {} '.format(math.sqrt(a*b*c)))

File: PF Liu/PF1.c
#include<stdio.h>
#include<math.h>

int main(void){
	int length,width,height;
	scanf("%d %d %d",&length,&width,&height);
	
	int a,b,c;
	a=length*width;
	b=width*height;
	c=length*height;
	
	printf("the surface area is %d\n",2*(a+b+c));
	printf("the volumn is %d\n",sqrt(a*b*c));
	
	return 0;
}
